cmr/queries.py

```diff
@@ -390,14 +442,14 @@ def convert_to_string(date: Optional[DateLike], default: datetime) -> str:
         # if we have both dates, make sure from isn't later than to
         if date_from and date_to and date_from > date_to:
             raise ValueError("date_from must be earlier than date_to.")
-        
+
         return date_from, date_to
 
     def revision_date(
-        self,
-        date_from: Optional[DateLike],
-        date_to: Optional[DateLike],
-        exclude_boundary: bool = False,
+            self,
+            date_from: Optional[DateLike],
+            date_to: Optional[DateLike],
+            exclude_boundary: bool = False,
```

Comment:
I think extra indentation slipped in via another PR:

```suggestion
        self,
        date_from: Optional[DateLike],
        date_to: Optional[DateLike],
        exclude_boundary: bool = False,
```